server/manager/src/main/java/org/apache/accumulo/manager/TabletGroupWatcher.java

```diff
@@ -663,7 +662,7 @@ private int assignedOrHosted(Map<TableId,TableCounts> last) {
     return result;
   }
 
-  // ELASITICITY_TODO: Remove
+  // ELASITICITY_TODO: Remove??
```

Comment:
If this is no longer called, which I think it isn't, then we can remove.

Reply:
I opened #3605 about this.  Need to understand why the merge code was adding a split to the table as part of the merge process.  This may have been related to chop compactions and ensuring deleted data was not brough back, however not completely sure.  I think we will kinda figure it out with the no chop merge work.  I will remove the code and add a reference to #3605 in the code where it was called.

Reply:
Decided to leave the uncalled method for now because it has a lot of stuff related to merge that I don't understand and need to research.  Mentioned the issue in comments in a9e48ea